src/foam/nanos/auth/User.js

```diff
@@ -106,9 +106,8 @@ emailIsSet_ = true;`
     },
     {
       class: 'FObjectProperty',
-      of: 'foam.nanos.auth.Address',
```

Comment:
it would be a good idea to keep this of property set to 'foam.nanos.auth.Address'

Reply:
Agreed.